### ISSUE TITLE
[기타] 맥에서 부계정 이용해 협업 테스트시 키체인 문제

### ISSUE THREAD

**Issue by librabit** [Cat-Hanbit/README#53]
## 어떤 유형의 이슈인가요?
기타

## 본문의 위치는 어디인가요?
1장 4절 142p

## 실습 환경을 알려주세요
Mac 하이 시에라 10.13.6, SourceTree 3.2.1)

## 세부 내용
1장 4절의 부계정 생성하여 포크 후 like.md 파일 생성하고 푸시하는 과정에서
커밋 계정을 변경하고 커밋 후 푸시하면 오류가 납니다.

이는 MacOS 의 KeyChain을 통해 깃헙에 접속해 푸시하는 과정에서
키체인에 저장된 내용이 본 계정의 아이디와 비밀번호로 시도하기 때문입니다.

해결하기 위해서는 키체인을 열고 깃허브 항목을 찾은 뒤 부계정의 아이디와 비밀번호를 입력해야 합니다.

![스샷 11](https://user-images.githubusercontent.com/42596258/94235729-f87da980-ff46-11ea-9c70-7d7ed8cb941a.png)

종류가 인터넷 패스워드 internet password 인 값을 찾아 열어두고, 계정을 바꿀 때마다 변경해줘야 가능하네요.

**Comment by honux77** [comment]
네 맞습니다. 키체인을 이용하는 방법도 있구요. 맥의 경우는 저장소 주소에 아이디@저장소주소 ( ex: https://cat-hanbit@github.com/cat-hanbit/readme) 이렇게 아이디를 쓰는 방법도 됩니다. 소스트리가 인증관련 기능이 좀 깔끔하지 않은 것 같네요. 개선중이라서인지 버전마다 조금씩 동작이 다릅니다.
